packages/services/server/src/graphql-handler.ts

```diff
@@ -163,15 +163,14 @@ export const graphqlHandler = (options: GraphQLHandlerOptions): RouteHandlerMeth
         });
 
         response.headers.forEach((value, key) => {
-          reply.header(key, value);
+          reply.header(key, value); // eslint-disable-line @typescript-eslint/no-floating-promises -- false positive, FastifyReply.then returns void
```

Comment:
this rule has some issues with `FastifyReply.then` type, more info https://github.com/typescript-eslint/typescript-eslint/issues/4961